bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -521,7 +521,10 @@ spec:
           export OVN_NB_DAEMON=$(ovn-nbctl --pidfile=/var/run/ovn/ovn-nbctl.pid \
             --detach \
             -p /ovn-cert/tls.key -c /ovn-cert/tls.crt -C /ovn-ca/ca-bundle.crt \
-            --db "{{.OVN_NB_DB_LIST}}")
+            --db "{{.OVN_NB_DB_LIST}}" --log-file=/run/ovn/ovn-nbctl.log)
+
+          # include nbctl daemon logging, allow for ovn-nbctl to create the log file
+          tail -f --retry /run/ovn/ovn-nbctl.log &
```

Comment:
why do we need to tail it?

Reply:
so we can see exactly in the ovnkube-master logs when the commands happen. Really helpful in debugging. I suspect as we migrate more and more to the golang bindings they will eventually go away, but it's a good way to track things as well that need to get migrated.

Reply:
Sorry I should have provided more context. I meant, yes, its useful to tail it, but that data is already in the log file on disk. But I suppose the real value add is the interleaving of those commands with the master logs.

Reply:
`-f --retry` is `--follow=descriptor --retry` whereas the usual `-F` is `-follow=name --retry`

```
       -f, --follow[={name|descriptor}]
              output appended data as the file grows;

              an absent option argument means 'descriptor'
```

We usually want `--follow=name`, is there a reason we are only following descriptor.

Also, do we want `--pid=$(cat /var/run/ovn/ovn-nbctl.pid)`  as in the other tail commands to stop the tail once the daemon stops?